resources/analysisTools/qcPipeline/bwaMemSortSlim.sh

```diff
@@ -137,6 +137,9 @@ ${SAMBAMBA_FLAGSTATS_BINARY} flagstat -t 1 "$NP_FLAGSTATS" > "$tempFlagstatsFile
 if [[ ${bamFileExists} == true ]]
 then
 	echo "the BAM file already exists, re-creating other output files."
+	if [[ "$runBwaPostAltJs" == "true" ]]; then
+		throw 50 "bwa-postalt.js script needs name-sorted input. Cannot run on existing (position sorted) BAM."
+	fi
```

Comment:
In some ways of running the Roddy, when the input files are automatically inferred from the input/output directories, there is a chance of an error. I am not sure what the bwa-postalt.js script does with name-sorted input, but I prefer to report this configuration error early (also because it will not be trivial to understand the cause of an error when debugging this JavaScript code).